pom.xml

```diff
@@ -4,7 +4,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>com.microsoft.azure</groupId>
 	<artifactId>azure-servicebus-jms</artifactId>
-	<version>1.0.1</version>
+	<version>2.0.0-beta.0</version>
```

Comment:
>2.0.0-beta.0< [](http://example.com/codeflow?start=10&length=13)

the convention that sdk team seems for beta versions is to got with a minor version 1. 

[https://mvnrepository.com/artifact/com.azure/azure-messaging-servicebus](https://mvnrepository.com/artifact/com.azure/azure-messaging-servicebus)

So lets also do the same and make it 2.0.0-beta.1